package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "glue",
   "description": "Server composer for hapi.js",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Comment:
@Marsup so better just to wait reaction from owner ? Or I can change request to make approval faster ? Thank you.